controllers/telegram_handlers.py

```diff
@@ -40,6 +90,19 @@ async def button_query_handler(update: Update, context: ContextTypes.DEFAULT_TYP
     if "class_schedule" in update.callback_query.data:
         await get_class_schedule_handler(update, context)
 
+    if "faculty_feedback" in update.callback_query.data:
```

Comment:
Shouldnt the /slash-commands for this be /faculty_feedback rather than /feedback…
Also add the /slash_commands for the wifi functions as well in readme

Reply:
We can use camel case instead cause it cause it's tedious to write an underscore

Reply:
Well, yah.
Can you do check if it works. It seems like it checks the string value as in the commands, maybe faculty_feedback should be converted to feedback here.

Reply:
camel case works. Pushed an update for this and some other commands that could use it